apis/s3/v1beta1/replicationConfiguration_types.go

```diff
@@ -206,7 +206,7 @@ type EncryptionConfiguration struct {
 	// supports symmetric customer managed CMKs. For more information, see Using
 	// Symmetric and Asymmetric Keys (https://docs.aws.amazon.com/kms/latest/developerguide/symmetric-asymmetric.html)
 	// in the AWS Key Management Service Developer Guide.
-	ReplicaKmsKeyID string `json:"replicaKmsKeyId"`
+	ReplicaKmsKeyID *string `json:"replicaKmsKeyId"`
```

Comment:
what is the reasoning behind these type changes. I see that some changed from pointer and some to a pointer.

Reply:
It's just what the new version of the SDK expects